eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java

```diff
@@ -698,6 +712,37 @@ protected void updateTagValues(EwonData data) {
             BasicScanclassHistorySet historySet =
                     new BasicScanclassHistorySet(providerName, "_exempt_", -1);
 
+            // Add AllRealtime tag under Ewon _config tag folder
+            if (!registeredEwons.contains(device))
+            {
+                // Create path to Ewon's all realtime tag
+                String ewonRealtimePath = buildTagPath(device + "/_config/AllRealtime");
+
+                // Add Ewon to registered Ewon list
+                registeredEwons.add(device);
+
+                // Register all realtime tag write handler with provider
+                provider.registerWriteHandler(ewonRealtimePath, new WriteHandler() {
+                    public QualityCode write(TagPath tagPath, Object o) {
+                        try {
+                            boolean realtimeEnabledBool = (boolean) o;
+                            if (realtimeEnabledBool) {
+                                realtimeEwons.add(device);
+                            }
+                            else {
+                                realtimeEwons.remove(device);
+                            }
+                            provider.updateValue(ewonRealtimePath, o, QualityCode.Good);
+                        } catch (Exception e) {
```

Comment:
I think adding this will make this have a little better error recovery functionality

```
 } catch (ClassCastException e) {
                            logger.error(
                                    "Writing Ewon AllRealtime tag has failed. Incorrect datatype",
                                    e);
                            provider.configureTag(ewonRealtimePath, DataType.Boolean);
                            provider.updateValue(ewonRealtimePath, Boolean.FALSE, QualityCode.Good);
                        } catch (Exception e) {
                            logger.error("Writing Ewon AllRealtime tag has failed.", e);
                        }
```

Reply:
When possible add the throwable in with the call to error(...).  Makes debugging a lot easier.

logger.error(String msg, Throwable t);

Reply:
forgot a period in that log message after datatype.

Reply:
updated